src/ui/mainformview.js

```diff
@@ -166,7 +184,7 @@ export default class MainFormView extends View {
 		const t = this.locale.t;
 
 		// Create equation input
-		const mathInput = new LabeledFieldView( this.locale, createLabeledInputText );
+		const mathInput = new LabeledFieldView( this.locale, createLabeledTextarea );
```

Comment:
bring back textarea that we lost when resolving conflicts: https://github.com/bethinkpl/ckeditor5-math/pull/9/files#diff-2efc953309779ec3dc248aac8e35081f80c768c7ad0c8ef8bdaea55cd044e156L179